components/molecule/field/src/index.js

```diff
@@ -72,7 +80,7 @@ const MoleculeField = ({
       )}
       <div className={CLASS_INPUT_CONTAINER}>
         {!inline && extendedChildren}
-        {(successText || errorText) && (
+        {(successText || errorText || alertText) && (
```

Comment:
What about creating a variable before the return called `const hasValidationText = successText || errorText || alertText` so we can reduce the cognitive load?

Reply:
In fact, we can check the value of `typeValidationText`! Thanks!